include/libdnf/module/module_sack.hpp

```diff
@@ -67,6 +67,19 @@ class ModuleSack {
     /// @since 5.0
     std::vector<std::string> get_default_profiles(std::string module_name, std::string module_stream);
 
+    /// @return `true` the module stream is enabled on the system, `false` otherwise.
+    /// @param name Name of the module
+    /// @param stream Name of the module stream
+    /// @since 5.0
+    // @replaces libdnf:ModulePackageContainer.hpp:method:ModulePackageContainer.isEnabled()
+    bool is_enabled(const std::string & name, const std::string & stream) const;
```

Comment:
@m-blaha @pkratoch Do you think that we need `in_enabled` and `is_disabled` on two placec - `ModuleItem` and `ModuleSack`? Or would be better to have more general method - get_state() - But one item can have multiple states at the same time - enabled, default, active, disabled ...

Reply:
We agreed that the `ModuleState` enum (AVAILABLE, DEFAULT, ENABLED, DISABLED) consists of mutually exclusive states and "active" will be entirely different. So, I think it would be better to have a method `get_state()` and then separate method `is_active()`. But maybe it might still make sense to ask if a whole module is disabled and that's something that would have to be in `ModuleSack`.

Reply:
My original intention was to have it only as a `ModuleItem` method. But then I realized then in dnf4 there is use case where only name and stream is known (https://github.com/rpm-software-management/libdnf/blob/891380467c2b9bcee0f52371ba78e4bfb1017b9e/libdnf/module/ModulePackageContainer.cpp#L1826) . I'm not sure whether this is also gonna be present in dnf5 implementation, though.

Reply:
Marek, thank you for finding the example when search for name string is required. It means lets keep it in ModuleSack

Reply:
From performance perspective - for module processing also get_state - make sense.

Reply:
How about removing those two commits with `is_enabled`/`is_disabled` methods from PR?  They are just examples of modules state usage. The crucial part here is overriding system state from `/etc/dnf/modules.d` directory which I need for future work.

Reply:
I see. And is there a reason to have `is_enabled()` if there is `get_state()`? Now that I think about it, it would make sense to me if there were just both `get_state(const std::string & name)` and `get_state(const std::string & name, const std::string & stream)`. I don't know where and how it's used, but maybe there could be just these `get_state()` methods until there is a reason for adding `is_enabled()`, `is_disabled()`?

Reply:
Sorry, I didn't notice your comment until I sent mine. Sounds good to me to remove them until they are needed.

Reply:
I also agree about removal.

Reply:
it's gone already :)